Formula/erigon.rb

```diff
@@ -23,10 +23,7 @@ class Erigon < Formula
   end
 
   depends_on "gcc" => :build
-  # Support for go 1.20 is merged upstream but not yet landed in a tag:
-  # https://github.com/ledgerwatch/erigon/pull/6848
-  # Remove on next release.
```

Comment:
May still be blocked due to https://github.com/prysmaticlabs/gohashtree/issues/6 on ARM. Will update comment if so.